text/0000-grid-identity.md

```diff
@@ -0,0 +1,718 @@
+- Feature Name: grid-identity
+- Start Date: 2020-12-08
+- RFC PR: (leave this empty)
+- Hyperledger Grid Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Grid Identity is a role-based access control system for managing permissions
```

Comment:
[nitpick] This looks to be useful identity management feature. Can we make a note of it in the RFC that this can be used for non-supplychain use cases as well.

Reply:
Are you talking about non-Grid use cases?

Reply:
Yes